npm/webpack-dev-server/src/makeWebpackConfig.ts

```diff
@@ -22,8 +22,8 @@ interface MakeWebpackConfigOptions extends CypressCTOptionsPluginOptions, UserWe
   isOpenMode: boolean
 }
 
-const mergePublicPath = (baseValue = '/', userValue = '/') => {
-  return path.join(baseValue, userValue, '/')
+const mergePublicPath = (baseValue = '/') => {
+  return path.join(baseValue, '/')
 }
```

Comment:
nit: This method doesn't make very much sense anymore. Just inline it into `getLazyCompilationWebpackConfig` (or rename it)

Reply:
done.Done